packages/e2e/local-network/scripts/update-pools-certificate.sh

```diff
@@ -129,7 +129,7 @@ for ((i = 0; i < ${#SP_NODES[@]}; ++i)); do
   cardano-cli stake-pool registration-certificate \
       --cold-verification-key-file "$coldVKey" \
       --vrf-verification-key-file "$vrfKey" \
-      --pool-pledge 500000000 \
+      --pool-pledge 0 \
```

Comment:
Could somebody please point me a good reading to understand the meaning of this change?
Thank you

Reply:
https://hydra.iohk.io/build/17427421/download/1/delegation_design_spec.pdf
https://github.com/input-output-hk/cardano-ledger